tests/conftest.py

```diff
@@ -539,19 +543,131 @@ def generate_sofa_postype_error(
 
 @pytest.fixture
 def views():
+    """ Used for the creation of Orientation objects with
+    `Orientations.from_view_up`
+    """
     return [[1, 0, 0], [2, 0, 0], [-1, 0, 0]]
 
 
 @pytest.fixture
 def ups():
+    """ Used for the creation of Orientation objects with
+    `Orientations.from_view_up`
+    """
     return [[0, 1, 0], [0, -2, 0], [0, 1, 0]]
 
 
 @pytest.fixture
 def positions():
+    """ Used for the visualization of Orientation objects with
+    `Orientations.show`
+    """
     return [[0, 0.5, 0], [0, -0.5, 0], [1, 1, 1]]
 
 
 @pytest.fixture
 def orientations(views, ups):
+    """ Orientations object uses fixtures `views` and `ups`.
+    """
     return Orientations.from_view_up(views, ups)
+
+
+@pytest.fixture
+def coordinates():
+    """ Coordinates object.
+    """
+    return Coordinates([0, 1], [2, 3], [4, 5])
+
+
+@pytest.fixture
+def signal():
```

Comment:
I would suggest rename it to something like `sine_signal` to indicate its properties